### PR TITLE
fix: add notices back to pricing page

### PR DESCRIPTION
**Pull request by jfcurran**
notices removed in #488 added back to `page-content/content-pricing.tsx` and `page/pricing.tsx`

### CODE REVIEW

**Review by BlueAcidFrog — APPROVED** [review]

**Review by aeneasr — CHANGES_REQUESTED** [review]
> 1 Upgrade from Developer plan to production plans possible, please contact us.

This does not make sense. People can now update to production in self service. Please remove this.

> 2 For a limited time the first sign ups to Start Up and Growth plans are free for one year.

How? It does not say how I can get it for free.

File: src/page-content/pricing-tiers/pricing-cloud.tsx [hunk 1]
```diff
@@ -67,15 +67,18 @@ export const pricingCloud: Pick<PricingProps, 'cloud'> = {
...
+          <span className={cn('is-themed-primary', 'is-semibold')}>
+            <sup> 2</sup>
+          </span>
         </>
       ),
       priceIsBilledMonthly: true,
       priceIsMonetary: true,
       button: {
         style: 'outlined',
         title: 'Buy now',
```
Comment (aeneasr) [inline src/page-content/pricing-tiers/pricing-cloud.tsx:80]: I just saw that this now links to the form again. I suggest to change this. Either have two buttons "Buy now" (links to console sign up) and "Get free access" (links to form). Or have one "Get free access" which links to the form, or something else. But as it stands right now, it's too confusing.

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: src/page-content/pricing-tiers/pricing-cloud.tsx (hunk 1 above)
Comment (BlueAcidFrog) [inline src/page-content/pricing-tiers/pricing-cloud.tsx:80]:
```suggestion
        title: 'Free access ',
```

**Comment by aeneasr** [comment]
Just changes two words to make it clearer and also ensure the date is obvious (1.3.2022 works for europe but not US).

**Comment by aeneasr** [comment]
Thank you all for the quick turnover!

**Pull request merged**